src/Contexts/Adaptors/SubstrateHomeAdaptor.tsx

```diff
@@ -201,6 +201,7 @@ export const SubstrateHomeAdaptorProvider = ({
               handleSetHomeChain(
                 homeChains.find((item) => item.type === "Substrate")?.chainId
               );
+              setIsReady(true);
```

Comment:
Please, add to the description the reason for this change.

Reply:
The reason is that after a transaction is finished and we click on `change` button the `isReady` variable is set to `false` and we need to make it `true` again.

Reply:
@AndreiNavoichyk, tested the backward flow from Cere to Polygon and it works fine.

Reply:
@AndreiNavoichyk, the problem is related to transfer because when you click "Start new transfer button" the destination bridge is reset and `disconnect` function becomes empty.

Reply:
Remove destination adaptor reset after transfer. It fixes the original reason of the issue.